Improve Operator Docs: Prerequisites and Current Testnet Parameters (#125)
oasislabs/docs: src/operators/prerequisites.md

repository's `go/` path. This binary contains both the logic for running an
Oasis node and also provides a CLI for handling registry and staking
operations.

### Building from source

Although highly suggested, building from source is currently beyond the scope
of this documentation. See [Oasis Core's building instructions][core-build] for
details.

::: warning
The code in the [`master` branch] might be incompatible with the code used by
other Nodes in the Public Testnet.

Make sure to use the version specified in the [Current Testnet Parameters][
params].
:::

[core-build]: https://github.com/oasislabs/oasis-core#developing-and-building-the-system
[`master` branch]: https://github.com/oasislabs/oasis-core/tree/master/
[params]: ./current-testnet-parameters.md

### Using inside Docker

For those that wish to use the Oasis-provided Docker image, the `oasis-node`
binary can be found inside the [oasis-node][oasis-node-docker] Docker image.

To start a Docker container from this image, set the appropriate version by
replacing `<DOCKER-IMAGE-VERSION>` with the version specified in the [Current
Testnet Parameters][params] and run:

```

> VERSION=<DOCKER-IMAGE-VERSION>

Reviewer: This should be either `export VERSION=<DOCKER-IMAGE-VERSION>` or `VERSION=<DOCKER-IMAGE-VERSION> docker run...` in the same line or
```
VERSION=<DOCKER-IMAGE-VERSION> \
docker run...
```
(notice the trailing \\).

I would go with `export` command, because we also need this env var in the next section for joining the testnet.

Reviewer 2: After this is resolved, I think we can go ahead and merge this PR.

Author: Hmm... I think `export` or "same line" is not necessary, e.g.:
```bash
[tadej@toronto ~]$ FOO=bar
[tadej@toronto ~]$ echo Test \
>   Multiline \
>   $FOO
Test Multiline bar
[tadej@toronto ~]$
```

Reviewer 2: Ah yeah, for this set of commands we only need a shell variable.

docker run -it --rm \
  --entrypoint /bin/bash \
  --volume $(pwd):/workdir \
  --workdir /workdir \
  oasislabs/oasis-node:$VERSION
```

This invocation will set `/workdir` in the Docker container to the current
working directory of the host.

::: tip NOTE
It is up to you to set this working directory correctly.
If there are any strange errors, this is a common source of problems.
:::

Inside the container, you can use the `oasis-node` command as you would
locally on your host.

::: warning
While the `latest` tag is available for the `oasis-node` Docker image, we
suggest using the tag specified in the [Current Testnet Parameters][params] of

> the form `master-YYYYMMDDHHMMSS` so that your node won't drift in case the

Reviewer 2: ```suggestion
the form `master-YYYYMMDDHHMMSS` to ensure that your node is compatible in case the
```

Author: Thanks, fixed.
